client/coral-admin/src/components/UserDetail.css

```diff
@@ -116,29 +116,50 @@
 .bulkActionGroup {
   height: 52px;
   background-color: #efefef;
-
+  padding: 0 0 0 10px;
+  display: flex;
   i {
     margin-right: 0;
   }
-
   .bulkAction {
     display: inline-block;
     width: 48px;
     height: 48px;
     transform: scale(.7);
     min-width: 0;
   }
-
   .bulkAction:last-child {
     margin-left: -10px;
   }
+  span {
```

Comment:
Can you make a classname out of this with a nice descriptive name?

Reply:
sure!